components/FAQ/QuestionAnswer.tsx

```diff
@@ -11,7 +12,17 @@ export default function QuestionAnswer({ index, question, answer }: Props) {
   return (
     <Box behind={index % 2 ? 'bg-iflightorange' : 'bg-ifpink'}>
       <div className="p-12">
-        <div className="font-extended text-[1.75rem] mb-4">{question}</div>
+        <div id={String(index)} className="font-extended text-[1.75rem] mb-4">
+          {question}{' '}
+          <a
+            className={styles.directLink}
+            aria-hidden="true"
+            href={'#' + index}
+            title="Direct link to question"
+          >
+            #
+          </a>{' '}
+        </div>
```

Comment:
I might pitch something more URL-friendly / memorable than just numbers here? Maybe something like: [slug](https://www.npmjs.com/package/slug) ?

```js
import slug from 'slug'
// [...] naive implementation
const slugged = slug(question.slice(0, 30))
```
Then replace this bit with
```suggestion
        <div id={slugged} className="font-extended text-[1.75rem] mb-4">
          {question}{' '}
          <a
            className={styles.directLink}
            aria-hidden="true"
            href={'#' + slugged}
            title="Direct link to question"
          >
            #
          </a>{' '}
        </div>
```
B/c in my mind it's much better if we can link something like https://testnet.ironfish.network/faq#what-are-the-minimum-hardware-requirements-for-running-a-node than something numeric, especially if we change the order of things inadvertently.

Reply:
Another, potentially option, upon reflection, would be to add manual ids to things, i.e. `id: "minimum-hardware-requirements"` to the source (that is, the raw `questions` in `page/faq.tsx`)